### PR TITLE
String updates for empty state 

### PR DESCRIPTION
**Pull request by Randy424**
Screen Shot 2021-03-28 at 3 15 39 PM

### CODE REVIEW

File: frontend/public/locales/en/bma.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "editBareMetalAsset.button.submit":"Apply",
 
   "bareMetalAsset.emptyState.title":"No bare metal assets found",
+  "bareMetalAsset.emptyState.subtitle": "<p>You don't have any bare metal assets.</p>Click the <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> button to create your resource",
```
Comment (showeimer) [inline frontend/public/locales/en/bma.json:32]: I think these don't need the paragraph separation `<p />`

Comment (Randy424) [reply frontend/public/locales/en/bma.json]: At first I thought I needed <p> to use <br>, then I noticed <p> will create a newline all by itself after closing...best to use <br> though.

**Comment by Randy424** [comment]
Recent changes to emptyState title:

Screen Shot 2021-03-29 at 2 53 59 PM

Screen Shot 2021-03-29 at 2 53 50 PM

Screen Shot 2021-03-29 at 2 53 55 PM

File: frontend/public/locales/en/bma.json [hunk 2]
```diff
@@ -28,7 +28,8 @@
   "editBareMetalAsset.title": "Edit bare metal asset",
   "editBareMetalAsset.button.submit":"Apply",
 
-  "bareMetalAsset.emptyState.title":"No bare metal assets found",
+  "bareMetalAsset.emptyState.title":"You don't have any bare metal assets.",
+  "bareMetalAsset.emptyState.subtitle": "Click the <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> button to create your resource",
```
Comment (Randy424) [inline frontend/public/locales/en/bma.json:32]: @showeimer 
I found that I was misinterpreting Joy's design. The newline she had was the space between the title and the message of the emptyState, I thought it was all part of the message. No need for a <br>.

**Review by showeimer — APPROVED** [review]

**Pull request merged**